extensions/ql-vscode/src/model-editor/modeling-events.ts

```diff
@@ -48,6 +48,15 @@ interface InProgressMethodsChangedEvent {
   readonly methods: ReadonlySet<string>;
 }
 
+interface RevealInModelEditorEvent {
+  dbUri: string;
+  method: Method;
+}
+
+interface FocusDbEvent {
```

Comment:
I'm wondering about the name for this event. It's focusing (opening) an existing modelt editor tab, right? If yes: it might be less confusing to name it something like 'FocusModelEditorEvent'?

Reply:
Good point! I'll rename.